.gitignore

```diff
@@ -29,3 +29,16 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+# Frontend artifacts and libs
+app/assets/builds/*
+node_modules
+
+# Platform/IDE-specific settings
+.DS_Store
+.idea
+.vscode
```

Comment:
Have you considered adding these to a local global ignore. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

If each dev uses a global git ignore for their IDE files then each project's gitignore doesn't need to include all the folders developers' IDEs might add.

Reply:
agreed - this is just what the rails template added, but agreed it's not really needed here